tests/unit/lint/lint-test.js

```diff
@@ -0,0 +1,5 @@
+import { module, test } from 'qunit'; // eslint-disable-line
+
+/**
+ * Linting CI tests will be injected here
+ */
```

Comment:
nevermind! I found how it targets: https://github.com/Shopify/js-buy-sdk/pull/197/files/a6e1c46364b24a070ddb008e8841d31b3a71884a#diff-e56633f72ecc521128b3db6586074d2cR47